cumulus_etl/__init__.py

```diff
@@ -1,3 +1,3 @@
 """Cumulus public entry point"""
 
-__version__ = "1.0.0"
+__version__ = "1.1.0"
```

Comment:
Finally trying to be more disciplined about version bumps, even though we don't do pip releases